engines/joomla/nucleus/particles/contentarray.html.twig

```diff
@@ -21,7 +21,12 @@
     {% set categories = joomla.finder('category', category_options).published(1).language().limit(0).find() %}
 
     {# Content Finder #}
-    {% set article_finder = joomla.finder('content').category(categories).published(1).language() %}
+    {% if filter.articles %}
+        {% set article_options = filter.articles ? {id: [filter.articles|split(',')]} : {} %}
```

Comment:
Alright, I see now.
Thanks for the explanation :wink: 

I just committed the changes you requested.